components/form/BannerSettings.vue

```diff
@@ -108,6 +108,12 @@ export default ({
             :disabled="isUiDisabled"
             :label="t('banner.text')"
           />
+          <p
+            v-if="bannerType === 'bannerConsent'"
```

Comment:
I would rather add a prop `footnote` and pass the value and then display conditionally the footnotes.
In this way it would be less type specific and more component API specific.

It would be obviously nice to keep it consistent with the dialog option.

Another point may be related to the use of logic in the markup, which should be better to be avoided.

Reply:
Been looking at the component `LabeledInput` and I am not comfortable in adding the footnote because I would need to create a new parent wrapper and I am really unsure if this won't break stuff somewhere else...

I would prefer to keep it as is. Will remove the logic and add it as a computed.

Reply:
Yeah sure, I meant to add a prop `footnote` in the component `BannerSettings`, so when you use it in `c/_cluster/settings/banners.vue` you can simply input it there:

```html
        <BannerSettings
          v-model="bannerVal"
          banner-type="bannerConsent"
          :mode="consentMode"
          :footnote="t('banner.consentFootnote')"
        />
```